### PR TITLE
A couple of changes

### PR DESCRIPTION
**Pull request by stefansundin**
Hey there again.

After having used the program for a couple of weeks, I decided to make some changes.

- The menu was taking way too long to load, to the point that it was a bad user experience. My workaround to this is to not clear the list of services when you bring up the list. Instead, I think the trade-off of seeing the old state is Ok. It still refreshes the list and you'll see the updated list within a few seconds.
  - This could possibly be improved slightly by immediately modifying the state when you click to start/stop a service. But I have zero swift skills, so I don't know how to do that.
- Added "Start all" and "Stop all" items.
- Moved the refresh indicator to the bottom.

Hope you like it!

### CODE REVIEW

**Comment by stefansundin** [comment]
I added the ability to restart services by revealing alternative menu items (with the Option key). It works great so well that I impressed myself.

Please give it a try @andrewn. :)

**Comment by andrewn** [comment]
Hi @stefansundin, this is great stuff! I'll give it a try soon and then create a new release.

File: BrewServicesMenubar/AppDelegate.swift [hunk 1]
```diff
@@ -90,33 +109,56 @@ class AppDelegate: NSObject, NSApplicationDelegate {
...
 
                 statusMenu.addItem(item)
+
+                let altItem = NSMenuItem.init(title: "Restart "+service.name, action: #selector(AppDelegate.handleRestartClick(_:)), keyEquivalent: "")
+                altItem.representedObject = service
+                altItem.state = item.state
+                altItem.isEnabled = item.isEnabled
+                altItem.isAlternate = true
+                altItem.isHidden = true
+                altItem.keyEquivalentModifierMask = NSAlternateKeyMask
```
Comment (stefansundin) [inline BrewServicesMenubar/AppDelegate.swift]: not sure if this line is actually needed...

**Comment by andrewn** [comment]
Thanks again for this, it's great!

I added a change so that when the command has finished executing, the list is queried again. This should mean that the list is accurate the next time it's opened.

I've created a new release.

**Comment by stefansundin** [comment]
Nice. 👍

It may be worth mentioning the option-key restart feature in the README as well.

**Pull request merged**